service/src/test/java/com/ge/predix/acs/service/policy/evaluation/PolicyEvaluationServiceTest.java

```diff
@@ -284,4 +319,60 @@ private BaseResource getResource() {
         return new Object[][] { { null, "s1", "a1" }, { "r1", null, "a1" }, { "r1", "s1", null }, };
     }
 
+    @DataProvider(name = "filterPolicySetsInvalidRequestDataProvider")
+    private Object[][] filterPolicySetsInvalidRequestDataProvider()
```

Comment:
Add a test case where 2 policy sets are provided and first one returns Effect.INDETERMINATE